tools/distpartitioning/parmetis_postprocess.py

```diff
@@ -0,0 +1,99 @@
+import os
```

Comment:
Let's establish the habit to always run isort and black.

Reply:
Done.